pkg/identityserver/gormstore/client.go

```diff
@@ -52,6 +52,8 @@ type Client struct {
 
 	Grants Grants `gorm:"type:INT ARRAY"`
 	Rights Rights `gorm:"type:INT ARRAY"`
+
+	TieAccessToSession bool `b	gorm:"not null"`
```

Comment:
```suggestion
	TieAccessToSession bool `gorm:"not null"`
```

Reply:
And please update `pkg/identityserver/storetest/client_store.go` to cover this new field.